nss-day-cohort-45/Hello-World-The-Animation-Durations
# adding England

## changes
1) added pic
2) added list items

## test
1) make sure pic is there
2) make sure all items are added to all lists
